starfish/pipeline/features/spot_attributes.py

```diff
@@ -11,7 +11,7 @@ class SpotAttributes(ValidatedTable):
     required_fields = {
         'x',  # spot x-coordinate
         'y',  # spot y-coordinate
-        'r',  # spot radius
```

Comment:
conflicts with `Features.round`

Reply:
this may break notebooks that try to refer to a spot's radius

Reply:
I'll make sure there aren't any missed references. 👍